1-exercises/J-string-substring/exercise2.js

```diff
@@ -14,11 +14,11 @@ let names = [
   "Arron Graham",
 ];
 
-names[0] = names[0].substring();
-names[1] = names[1].substring();
-names[2] = names[2].substring();
-names[3] = names[3].substring();
-names[4] = names[4].substring();
+names[0] = names[0].substring(0,names[0].length-8);
+names[1] = names[1].substring(0, names[1].length - 5);
+names[2] = names[2].substring(0, names[2].length - 8);
+names[3] = names[3].substring(0, names[3].length - 7);
+names[4] = names[4].substring(0, names[4].length - 7);
 
```

Comment:
I'm not sure of the purpose of this exercise, but nothing more than this is needed:
```javacript
names[0] = names[0].substring(0, 6);
```